src/commands/deployContainerApp/deployContainerApp.ts

```diff
@@ -56,18 +57,22 @@ export async function deployContainerApp(context: IActionContext, node?: Contain
     }
     wizardContext.telemetry.properties.revisionMode = item.containerApp.revisionsMode;
 
+    const confirmationViewTitle: string = localize('summary', 'Summary');
+    const confirmationViewDescription: string = localize('viewDescription', 'Please select an input you would like to change. Otherwise click "Confirm" to deploy.');
+    const title: string = localize('deployContainerAppTitle', 'Deploy image to container app');
+
     const wizard: AzureWizard<ContainerAppDeployContext> = new AzureWizard(wizardContext, {
-        title: localize('deployContainerAppTitle', 'Deploy image to container app'),
+        title: title,
```

Comment:
nit: Is it possible to have the title and the editor tab title to be different?
![{593E4551-7631-4915-BEAE-9AF573BE079B}](https://github.com/user-attachments/assets/1841d359-a021-4bd8-856c-0ff313c32b71)
While I prefer summary for the header title, I think it just saying "Summary" in the VS Code tab doesn't look as good.

Reply:
Hmm, could we have the text here (and maybe the header) based on the command? Ex: "Deploy image to container app - Summary"